src/appdev/ApplicationRouter.ts

```diff
@@ -94,7 +98,7 @@ class ApplicationRouter<T> {
    * for the given request.
    */
   async content(req: Request): Promise<T> {
-    throw new Error(null);
+    throw new Error('Error');
```

Comment:
Does this error get thrown if this function isn't overriden? If so, we should have the error message reflect that rather than having 'Error'

Reply:
I didn't actually read through what the AppDev Application Router does, but I changed this so it function the same as it used to.